system/Email/Email.php

```diff
@@ -760,43 +763,43 @@ protected function stringToArray($email)
     /**
      * @param string $str
      *
-     * @return Email
+     * @return $this
      */
     public function setAltMessage($str)
     {
-        $this->altMessage = (string) $str;
+        $this->altMessage = $str;
 
         return $this;
     }
 
     /**
      * @param string $type
      *
-     * @return Email
+     * @return $this
      */
     public function setMailType($type = 'text')
     {
-        $this->mailType = ($type === 'html') ? 'html' : 'text';
+        $this->mailType = $type === 'html' ? 'html' : 'text';
 
         return $this;
     }
 
     /**
      * @param bool $wordWrap
      *
-     * @return Email
+     * @return $this
      */
     public function setWordWrap($wordWrap = true)
     {
-        $this->wordWrap = (bool) $wordWrap;
+        $this->wordWrap = $wordWrap;
```

Comment:
Why are we getting rid of casting here?

Reply:
Because it is already `bool` in the PHPDoc.

Reply:
Returned to previous.